blueprints/soci-snapshotter/README.md

```diff
@@ -0,0 +1,149 @@
+
+# Karpenter Blueprint: Using SOCI snapshotter parallel pull/unpack mode
+
+## Purpose
+
+SOCI parallel pull/unpack mode accelerates container image loading through concurrent downloads and decompression, reducing image pull time by up to 50% for large container images.
+
+This optimization is essential when you need complete image availability before container startup but want to eliminate the performance bottleneck of sequential layer downloading. While lazy loading optimizes for immediate startup, parallel pull/unpack mode maximizes throughput during traditional image pulls - making it ideal for batch workloads, AI/ML applications with large images, and high-performance environments where fast, complete image loading is critical.
```

Comment:
maybe we put this first? blueprints need to start with a very clear problem and how the blueprint will guide you to solve it ... the info is already here, we just need to order it a bit.